src/helper/style-path.js

```diff
@@ -465,11 +465,23 @@ const getColorsFromSelection = function (selectedItems, bitmapMode) {
 
                     let strokeColorString = primary;
                     const strokeColor2String = secondary;
-                    const strokeGradientType = gradientType;
+                    let strokeGradientType = gradientType;
+
+                    if (strokeGradientType !== GradientTypes.SOLID &&
+                        item.strokeColor.gradient.stops.length === 2 &&
+                        item.strokeColor.gradient.stops[0].color.alpha === 0 &&
+                        item.strokeColor.gradient.stops[1].color.alpha === 0) {
+                        // Clear the gradient if both colors are transparent
+                        item.strokeColor = null;
```

Comment:
I feel like a function which *retrieves* color state from shapes should not modify those shapes' colors directly